### DISCUSSION TITLE
Change Username

### DISCUSSION THREAD

**Discussion by hanried** [earthlab-education/Earth-Analytics-AY24#110] [Q&A]
Hi! I originally created my GitHub account a couple years ago when dabbling in a asynchronous coding course. I don't remember the details behind creating my username, however now I think the username is kind of confusing. It's @reidhan, but my last name is spelled Rieder. I think it's confusing that my username sort of contains my last name but it's spelled wrong. 

Is it a bad idea to change my username at this point? I was reading through [this help article ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) and I don't think changing my username will have huge implications in what I've done so far in this course, but I'm not sure. Thanks in advance for any thoughts!

**Comment by eculler** [comment]
Hi @reidhan you can definitely go for it. If you lose access to anything, send me a message or ask your section leader and we'll get you set up again.

One thing this will change is the location of any GitHub Pages you have, so be sure to update the link to your portfolio if you've posted it anywhere so far.

**Reply by hanried** [reply]
Thank you!! I was able to update my username. I updated my info in the README.md file in my branch of our Earth-Analytics-AY24 repo. I committed the changes and made a pull request.
It looks like my branch name and the issues assigned to me in the Earth-Analytics-AY24 repo still have my old username. Do those things take longer to update? Or do I need to do something specific to update those items?
